src/components/CollectionDetailPage.vue

```diff
@@ -250,21 +259,20 @@ export default {
       });
     },
     remove(collection) {
-      const sure = confirm(this.$t('confirm_delete', [collection.name]));
-      if (sure) {
-        this.$store.dispatch('collections/DELETE_COLLECTION', collection.uid).then(() => {
-          this.fetch();
```

Comment:
Hi @PaulSchroeder there is no need to do this.fetch() inside a Promise :) even the then is useless if yo're not doing anything inside the callback function

Reply:
the fetch is red, means I removed it  ;)
I still need `then` to update the collection list, e.g. remove the deleted item from the list of collections.

Reply:
eheheh apologies!!!!